README.md

```diff
@@ -52,11 +53,7 @@ For example, to launch Apple Music in WSA, use:
 wsa://com.apple.android.music
 ```
 
-Some URIs are not supported in WSA, such as:
-
-```shell
-wsa://com.android.settings
-```
+Some URIs may not be supported in older versions of WSA.
 
```

Comment:
What's the thinking behind this change? This URI still works.

Reply:
In master branch, the note mentions:
Some URIs are not supported in WSA, such as:
```
wsa://com.android.settings
```
However, this URI is supported in Android 12 version of WSA, so running the URI from Run dialog opens Android Settings app.